.github/workflows/ci-cd.yml

```
@@ -587,18 +587,214 @@
+                  'speed-index': ['warn', { maxNumericValue: 4000 }]
+                }
+              },
+              upload: {
+                target: 'temporary-public-storage'
+              }
+            }
+          };
+          EOF
+
+      - name: Wait for deployment to be fully ready
+        run: |
```

actionlint (1.7.7)

property "deploy-production" is not defined in object type {}

(expression)